source/making_payments/index.html.md.erb

```diff
@@ -38,7 +38,9 @@ For example:
 
 The amount must be a number data type rather than a string.
 
-The minimum amount is one pence. The maximum amount is 10,000,000 pence (£100,000). [Contact us](/support_contact_and_more_information/#contact-us) if you need to take payments above the maximum amount.
+The minimum amount is one pence plus your Payment Service Provider's (PSP's) transaction fee. Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) and check your contract to see your PSP's transaction fee.
+
+The maximum amount is 10,000,000 pence (£100,000). [Contact us](/support_contact_and_more_information/#contact-us) if you need to take payments above the maximum amount.
```

Comment:
As discussed, let's add some info about minimum payouts.  ie. you have to have a minimum amount owed to you before your PSP will deposit it into your bank account. For Stripe it's £1 - so if a service does a test payment of 20p, it will go through and is above the min including transaction fees, but they won't actually see it deposited into their bank account. If they are trying to check the full end to end process, that might confuse them.